src/variableinspector.ts

```diff
@@ -133,6 +150,9 @@ export
     }
 
     protected onInspectorUpdate( sender: any, allArgs: IVariableInspector.IVariableInspectorUpdate): void {
+        if (!this.isAttached) {

```

Comment:
A simple way for now to avoid creating the table when it is not attached.

We should check whether it is necessary to recreate the table when the panel is not attached to the main area (it looks like `onInspectorUpdate`  is triggered on cell execute).

Reply:
Good catch!